connectors/sources/jira.py

```diff
@@ -65,7 +66,8 @@
     ATTACHMENT_CLOUD: "/rest/api/2/attachment/content/{attachment_id}",
     ATTACHMENT_SERVER: "/secure/attachment/{attachment_id}/{attachment_name}",
     USERS: "/rest/api/3/users/search",
-    PROJECT_PERMISSIONS_BY_KEY: "/rest/api/2/user/permission/search?projectKey={project_key}&permissions=BROWSE_PROJECTS&maxResults={max_results}&startAt={start_at}",
+    USERS_FOR_DATA_CENTER: "/rest/api/latest/user/search?username=''&startAt={start_at}&maxResults={max_results}",  # we can fetch only 1000 users for jira data center
```

Comment:
Is there some public source for this limitation we can add here?

Reply:
There is no concrete discussion over this limitation, but it includes in this community post - https://community.atlassian.com/t5/Jira-Software-questions/Get-a-list-of-all-users-using-API-for-data-center/qaq-p/2401657. Do you think we should add this in the code?